scipy/spatial/transform/rotation.pyx

```diff
@@ -878,22 +878,27 @@ cdef class Rotation(object):
         Parameters
         ----------
         seq : string
-            Specifies sequence of axes for rotations. Up to 3 characters
+            Specifies the order of the sequence of
+            rotations to be performed. Up to 3 characters
             belonging to the set {'X', 'Y', 'Z'} for intrinsic rotations, or
             {'x', 'y', 'z'} for extrinsic rotations. Extrinsic and intrinsic
             rotations cannot be mixed in one function call.
         angles : float or array_like, shape (N,) or (N, [1 or 2 or 3])
-            Euler angles specified in radians (`degrees` is False) or degrees
-            (`degrees` is True).
-            For a single character `seq`, `angles` can be:
+            where the number of rows, N, indicates the number of rotation 
```

Comment:
Three things here.

1. Generally I think it's a good addition to clarify how angles are arranged.
1. I think it's better not to start the description with "where ...". So put it after the original sentence.
2. "provide rotations angles about the z, y and x axes respectively for each set." This sounds misleading - why "z, y and x" in particular? Angles are matched to the corresponding axes given in `seq`.

Could you rework this?

Reply:
@valschmidt Not sure if you're planning to come back to this before our upcoming branch for 1.6.0 in < two weeks; I can bump the milestone if not.